component/src/main/java/org/wso2/extension/siddhi/gpl/execution/pmml/util/PMMLUtil.java

```diff
@@ -0,0 +1,51 @@
+package org.wso2.extension.siddhi.gpl.execution.pmml.util;
```

Comment:
where is the license header ??

Reply:
Added license in commit [#5055bb925fa7c0b2f096d90b1363f2c2b29a2c2a](https://github.com/wso2-extensions/siddhi-gpl-execution-pmml/pull/32/commits/5055bb925fa7c0b2f096d90b1363f2c2b29a2c2a)